src/Makefile.in

```diff
@@ -342,6 +376,7 @@ install-stratcond-headers:	$(STRATCON_HEADERS) install-headers
 install-headers:	$(HEADERS)
 	$(top_srcdir)/buildtools/mkinstalldirs $(DESTDIR)$(bindir)
 	$(top_srcdir)/buildtools/mkinstalldirs $(DESTDIR)$(includedir)
+	$(top_srcdir)/buildtools/mkinstalldirs $(DESTDIR)$(includedir)/flatbuffers
```

Comment:
I'd also like to install the `%.fbs` files, to make it simpler for other languages to generate messages that interact with noit flatbuffers.